pkg/operators/filter/filter.go

```diff
@@ -101,6 +100,32 @@ func (f *filterOperator) InstantiateDataOperator(gadgetCtx operators.GadgetConte
 	return fop, nil
 }
 
+func splitFilter(filterCfg string) []string {
```

Comment:
I think this func should be moved to the api package as it can potentially be reused a lot.

Reply:
Done. Let me know how it looks now!